tools/qontract_cli.py

```diff
@@ -4027,5 +4030,95 @@ def request_reconciliation(
             logging.info("External Resource does not exist")
 
 
+def _get_er_logs_grafana_url(key: ExternalResourceKey) -> str:
+    r = Reconciliation(key=key)
+    j = ReconciliationK8sJob(reconciliation=r)
+
+    panes = {
+        "dummy": {
+            "datasource": "P1A97A9592CB7F392",
+            "queries": [
+                {
+                    "id": j.name(),
+                    "region": "default",
+                    "namespace": "",
+                    "refId": "A",
+                    "datasource": {"type": "cloudwatch", "uid": "P1A97A9592CB7F392"},
+                    "queryMode": "Logs",
+                    "logGroups": [
+                        {
+                            "arn": "arn:aws:logs:us-east-1:744086762512:log-group:appsrep09ue1.external-resources-jobs:*",
+                            "name": "appsrep09ue1.external-resources-jobs",
+                            "accountId": "744086762512",
+                        }
+                    ],
+                    "expression": f"fields message | filter kubernetes.pod_name like /{j.name()}/\n",
+                    "statsGroups": [],
+                }
+            ],
+            "range": {"from": "now-24h", "to": "now"},
+        }
+    }
+    pane_s = urllib.parse.quote_plus(json.dumps(panes))
+    return f"https://grafana.app-sre.devshift.net/explore?schemaVersion=1&panes={pane_s}&orgId=1"
```

Comment:
instead of an explore url, can we build a dashboard for it? Can contain both logs and metrics.

Reply:
I tried it but log visualizations with Cloudwatch don't work very well. 
Especially this one: https://github.com/grafana/grafana/issues/24603 


I think having both is not exclusive, we can add another link column to the table